migrations/20210421034411-create-plant.js

```diff
@@ -8,9 +8,6 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
-      GardenId: {
```

Comment:
Sorry I missed this before, but if you go back to your Data Model diagrams in Figma, you'll notice that GardenId is NOT part of the Plant model.

That's because Plant and Garden have a MANY-TO-MANY relationship with each other, implemented by the GardenPlant JOIN TABLE...

So I'm removing it here.